### PR TITLE
Parse join reason and alert relevant people + Adds prettier workflow + Runs Prettier

### LINKED ISSUE #19: Parse join reason and alert relevant people

### PR DESCRIPTION
**Pull request by aramshiva**
Fixes #19
Related with #20

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
```
Comment (jaspermayone) [inline .github/workflows/CI.yml:1]: can you name this ci plz?

**Review by jaspermayone — DISMISSED** [review]
LGTM, wait to merge until #20 lands.

Also needs to be tested!

File: .github/workflows/CI.yml (hunk 1 above)
Comment (aramshiva) [reply .github/workflows/CI.yml:1]:
```suggestion
name: CI
```

**Review by jaspermayone — CHANGES_REQUESTED** [review]
MERGE CONFLICTS!

**Review by jaspermayone — APPROVED** [review]

**Pull request merged**